float64 distance
float64 theta
uint8 direction

uint8 LEFT = 0
uint8 CENTER = 1
uint8 RIGHT = 2